monitor.py

```diff
@@ -67,6 +67,7 @@ class EDLogs(FileSystemEventHandler):  # type: ignore # See below
     _RE_CANONICALISE = re.compile(r'\$(.+)_name;')
     _RE_CATEGORY = re.compile(r'\$MICRORESOURCE_CATEGORY_(.+);')
     _RE_LOGFILE = re.compile(r'^Journal(Alpha|Beta)?\.[0-9]{12}\.[0-9]{2}\.log$')
+    _RE_SHIP_ONFOOT = re.compile(r'^(FlightSuit|UtilitySuit_Class.)$')
```

Comment:
Is it confirmed whether or not `UtilitySuit_Class` is non-decimal? if its always a decimal this should be \d

Reply:
It's `"name": "UtilitySuit_Class1"` for the Maverick suit, but I didn't want to assume it'd just be digits.

What we need is a new Journal doc specifying what those can be, or to be far enough into the Alpha to try out other suits.  Do NOT assume Frontier are going to be consistent on such.  Even 'single character' might turn out to not be enough.

So it'll do for now.  Change it as we learn more.

Reply:
sounds good